backend/app/api/v1/commons/oso.py

```diff
@@ -0,0 +1,57 @@
+from datetime import date
+
+import pandas as pd
+
+from app.api.v1.commons.constants import HCE_FIELD_CONSTANT_DICT
```

Comment:
This doesn't appear to be referenced ... which is likely good as the "HCE" name isn't really appropriate here. 😁